src/components/atoms/ResultCountLabel/index.tsx

```diff
@@ -2,11 +2,14 @@ import { FC } from "react";
 
 import * as S from "./styles";
 import { ResultCountWrapperProps } from "./interface";
+import { white, primaryBackground } from "../../../assets/styles/colors";
 
 const ResultCountLabel: FC<ResultCountWrapperProps> = ({ count }) => {
   return (
     <S.ResultCountWrapper>
-      <S.ResultCountP>Wyniki wyszukiwania ({count})</S.ResultCountP>
+      <S.ResultCountP backgroundColor={count === 0 ? white : primaryBackground}>
```

Comment:
I cannot see any difference in the background colour when count is 0 or non 0. 🤔

Reply:
According to design, background colour changes only on mobile. For tablets and wider screens there is no difference.

Reply:
Yes, but why should it depend on the count? If I understand it correctly it should be connected with the viewport size (mobile).

Reply:
Nvm, I've misread the design. ;)